### DISCUSSION TITLE
How to increase build-time limit beyond 15 minutes.

### DISCUSSION THREAD

**Discussion by brentarias** [community/community#52565] [Enterprise]
### Select Topic Area

Question

### Body

I have a Github Enterprise subscription.  In that, I have a repo with CI/CD github actions which builds a small Angular app.  Github terminates the build at 15 minutes, and there is zero documentation from either github or 3rd parties of how to fix this.

- Do I need a different subscription?  Which one?  The subscription documentation only talks about monthly build minutes...and doesn't mention anything about a time-limit on individual builds.
- Do I need to adjust something in my CI/CD script?  The documentation simply says there is 360 minute build limit...which obviously isn't true (since my builds are cut-off in 15 minutes)...and no documentation on how to alter that limit.

**Comment by airtower-luna** [comment]
Does your workflow (or any action/reusable workflow it uses) define a limit? That's the only reason I can think of for such a low limit. [`timeout-minutes`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepstimeout-minutes) is the relevant setting on a step or job. If it's not *exactly* 15 min, did you maybe limit concurrency, and the job is canceled because of the next push?

I think for more it'd be necessary to see your workflow.
